.circleci/config.yml

```diff
@@ -88,6 +88,12 @@ jobs:
             done
 
             docker-compose up -d dks-standalone-http dks-standalone-https
+
+            while ! docker exec dks-standalone-http cat logs/dks.out | fgrep -q "Started DataKeyServiceApplication"; do
+              echo "Waiting for dks"
+              sleep 2
+            done
```

Comment:
you might want an exit clause like: if time > max_time; exit 1

Reply:
I think I should just remove the echo? If a job has no output for 10 minutes circle kills it. Failing that it kills it after 5 hours.

Reply:
If CircleCI has a 10 minute limit then that's a good enough exit clause